src/components/EntityInfo/_entity-info.scss

```diff
@@ -6,4 +6,8 @@
       grid-template-columns: repeat(2, 1fr);
     }
   }
+
+  &__grid-item {
+    overflow-wrap: anywhere;
```

Comment:
TIL that `overflow-wrap` is the official name for the commonly used (by me) `word-wrap` rule :)